app/services/create_batch.rb

```diff
@@ -1,6 +1,8 @@
 class CreateBatch
   def call
-    unprocessed_appointment_summaries = AppointmentSummary.unprocessed.where(format_preference: :standard)
+    unprocessed_appointment_summaries = AppointmentSummary.unprocessed
+                                        .where(format_preference: :standard)
+                                        .where(country: [nil, '', 'United Kingdom'])
```

Comment:
Yeah, here we've a scope with 2 chained `where`s, definitely a cause for concern